Dockerfile

```diff
@@ -1,3 +1,11 @@
+FROM golang:latest as builder
+
+ENV GOPATH=/build
+
+RUN go get github.com/itzg/go-mc-status
+
+#################################################
+
```

Comment:
Rather than building it from scratch, please add a `RUN easy-add` entry like those below to pull https://github.com/itzg/mc-monitor/releases/download/0.5.0/mc-monitor_0.5.0_linux_amd64.tar.gz

Reply:
👍